gatsby-config.js

```diff
@@ -25,7 +25,7 @@ const apiUrl =
     ? `https://api.github.com/graphql`
     : `https://git.corp.adobe.com/api/graphql`
 
-const patterns = process.env.GATSBY_SOURCE_PATTERNS.split(",")
+const patterns = process.env.GATSBY_SOURCE_PATTERNS.replace(/ /g,'').split(",")
```

Comment:
If a folder has a space in the name this will break the pattern matching. It would be better to split the GATSBY_SOURCE_PATTERNS envar then use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) to remove the leading and trailing spaces from each pattern.